docs-users/support/faq.md

```diff
@@ -46,3 +46,42 @@ With macOS, replace `Ctrl` by `Cmd`.
 * `mycolumn<12.34` → will match features whose column `mycolumn` is lower than `12.34` (as number)
 
 When the condition match, the associated style will be applied to the corresponding feature.
+
+
+## How to use variables ?
+
+In general, using a variable is as simple as `{myvar}`.
+
+It's possible to define another variable as fallback of the first one like this: `{myvar|fallbackvar}`.
+
+To fallback to a string, add it between double quotes: `{myvar|"fallback"}`.
+
+It's possible to combine more variables: `{myvar|othervar|"some string"}`.
+
+It's possible to use a variable inside an URL, for example: `[[https://domain.org/?locale={locale}|Wikipedia]]`.
+
+Or even as source for an image: `{{{myvar}}}` (note the triple `{}`).
+
+### Available variables in text fields:
+
+Those variables can be used in a feature description, or in popup content templates.
+
+Any property of the feature will be available, plus:
+- `{lat}/{lng}` → the feature position (or centroid in case of line or polygon)
```

Comment:
In the test we have `{lon}`, is that an error? (same below)

Reply:
No, we support both, but though to only document one, to keep the doc simple. Should I add `lon` to ?

Reply:
I'm OK with only documenting this one 👍